### DISCUSSION TITLE
Archive and search

### DISCUSSION THREAD

**Discussion by hgpit** [requarks/wiki#5316] [Help / Questions] [answered]
Hello,

Is it possible to do the following in Wiki.js - if not, would you consider including it in a future release?

We would like be able to 'archive' articles, which are no longer relevant to our internal day-to-day operations. This would mean that the article still exists, but is 'put aside' and not included in searches, for instance.

Obviously, sometimes we may want to search old articles, so a tick-box in the global search field/box would be able include such articles.

Very interested to hear any feedback.

Thank you!

Elliot :)

**Comment by hgpit** [comment]
Still would really love a solution for this :)

**Comment by NGPixel** [comment, marked as answer]
Submit this idea on https://requarks.canny.io/wiki
